src/libs/ajax/workflows-app/Cbas.test.js

```diff
@@ -0,0 +1,337 @@
+import 'setimmediate';
+
+import { MatchersV3, PactV3, SpecificationVersion } from '@pact-foundation/pact';
+import path from 'path';
+import { fetchFromProxy, fetchOk, jsonBody } from 'src/libs/ajax/ajax-common';
+import { Cbas } from 'src/libs/ajax/workflows-app/Cbas';
+import {
+  runSetInputDef,
+  runSetInputDefWithSourceNone,
+  runSetInputDefWithStruct,
+  runSetOutputDef,
+} from 'src/libs/ajax/workflows-app/CbasMockResponses';
+
+jest.mock('src/libs/ajax/ajax-common');
```

Comment:
Because we're using `jsonBody` directly in this file we shouldn't mock it out.
```suggestion
jest.mock('src/libs/ajax/ajax-common', () => ({
  ...jest.requireActual('src/libs/ajax/ajax-common'),
  fetchFromProxy: jest.fn(),
  fetchOk: jest.fn(),
  authOpts: jest.fn(),
}));
```